api/services/MachineService.js

```diff
@@ -291,7 +312,9 @@ function _createMachine(image) {
       return machine.getPassword()
         .then((password) => {
           machine.password = password;
-
+          // If machine have been assigned when booting we have to keep endDate and user
```

Comment:
If the machine have been assigned when booting `endDate` and `user` have been set in data base. We don't want to loose these informations. `endDate` and `user` `machine`'s attributes here are empty, we don't want erase these values when we will do an update of machine.